server/util.go

```diff
@@ -354,6 +354,7 @@ func AddCommands[T types.Application](rootCmd *cobra.Command, appCreator types.A
 		cometCmd,
 		version.NewVersionCommand(),
 		NewRollbackCmd(appCreator),
+		ModuleHashByHeightQuery(appCreator),
```

Comment:
This adds the command to the root context which makes it very prominent.
I don't have a strong opinion on this. `Debug` could be an option, too.

Reply:
I did this because including it in debug would require a change in the API of Cmd() (we would need to include appCreator as an input param). If this is desired I would be happy to do so.